From 078c0db2bc3adf7c665190ee7a9263d7fd1ec618 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Thu, 29 Apr 2021 13:06:12 +0200
Subject: [PATCH] gudevenumtypes: make deterministic

@filename@ writes the full path to the generated file,
which can vary in CI systems and make outputs non-reproducible.

Upstream-Status: Submitted [https://gitlab.gnome.org/GNOME/libgudev/-/merge_requests/15]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 gudev/gudevenumtypes.c.template | 2 +-
 gudev/gudevenumtypes.h.template | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/gudev/gudevenumtypes.c.template b/gudev/gudevenumtypes.c.template
index fc30b39..4918f7e 100644
--- a/gudev/gudevenumtypes.c.template
+++ b/gudev/gudevenumtypes.c.template
@@ -4,7 +4,7 @@
 /*** END file-header ***/
 
 /*** BEGIN file-production ***/
-/* enumerations from "@filename@" */
+/* enumerations from "@basename@" */
 /*** END file-production ***/
 
 /*** BEGIN value-header ***/
diff --git a/gudev/gudevenumtypes.h.template b/gudev/gudevenumtypes.h.template
index d0ab339..79bf7c4 100644
--- a/gudev/gudevenumtypes.h.template
+++ b/gudev/gudevenumtypes.h.template
@@ -9,7 +9,7 @@ G_BEGIN_DECLS
 
 /*** BEGIN file-production ***/
 
-/* enumerations from "@filename@" */
+/* enumerations from "@basename@" */
 /*** END file-production ***/
 
 /*** BEGIN value-header ***/
-- 
2.30.2

